M2/include/configuration.in

```diff
@@ -81,6 +81,8 @@ NODENAME=@NODENAME@
 OBJDUMP=@OBJDUMP@
 OBJEXT=@OBJEXT@
 OPTIMIZE=@OPTIMIZE@
+OPENMP_CXXFLAGS=@OPENMP_CXXFLAGS@
```

Comment:
Let's not use variables with these two names, as they are not needed.

Reply:
Then I don't understand the logic of the (rewritten) AC_OPENMP macro. Isn't the effect of the macro to populate these variables in line 57:
`OPENMP_[]_AC_LANG_PREFIX[]FLAGS=$ac_cv_prog_[]_AC_LANG_ABBREV[]_openmp ;;`
?

Reply:
Oops, good point!  But then let's use them closer to the use of the macro, by changing
```
AC_SUBST(OPENMP_CXXFLAGS,"$OPENMP_CXXFLAGS")
AC_SUBST(OPENMP_LDFLAGS,"")
```
to
```
CXXFLAGS="$CXXFLAGS $OPENMP_CXXFLAGS"
LDFLAGS="$LDFLAGS $OPENMP_CXXFLAGS"
```
It's good to keep mysteries localized.

Reply:
I think it does not work like this. I'm pushing a commit in a minute where I implemented this solution, but then compiling gc fails during the linking stage. I think adding -fopenmp to the LDFLAGS is not OK with Apple clang.